### ISSUE TITLE
Should we use shelljs for the scripts?

### ISSUE THREAD

**Issue by nikeshnazareth** [weare2brothers/overcooked-cms#27]
We've had issues with cross-platform scripts, where commands that run on
a mac do not work on windows.

Shelljs ( https://github.com/shelljs/shelljs ) attempts to address this,
but it seems to have a limited set of functions and is more overhead.

Not sure if it's worth it.

What do you think @bigfriedicecream ?

**Comment by bigfriedicecream** [comment]
Yeah, overhead is annoying, but don't mind using it if it does everything we need.

The only other option I can think of at this point would be to add scripts in `package.json` to target windows and mac.

Perhaps we can try it out when we're doing more cms work next.
